zrml/global-disputes/src/weights.rs

```diff
@@ -0,0 +1,92 @@
+//! Autogenerated weights for zrml_global_disputes
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-08-30, STEPS: `20`, REPEAT: 50, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/zeitgeist
+// benchmark
+// pallet
+// --chain=dev
+// --steps=20
+// --repeat=50
+// --pallet=zrml_global_disputes
+// --extrinsic=*
+// --execution=wasm
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --template=./misc/weight_template.hbs
+// --output=./zrml/global-disputes/src/weights.rs
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use core::marker::PhantomData;
+use frame_support::{traits::Get, weights::Weight};
+
+///  Trait containing the required functions for weight retrival within
+/// zrml_global_disputes (automatically generated)
+pub trait WeightInfoZeitgeist {
+    fn vote_on_outcome(o: u32, v: u32) -> Weight;
+    fn unlock_vote_balance(l: u32, o: u32) -> Weight;
+    fn add_vote_outcome(w: u32) -> Weight;
+    fn reward_outcome_owner(o: u32, k: u32) -> Weight;
+}
+
+/// Weight functions for zrml_global_disputes (automatically generated)
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfoZeitgeist for WeightInfo<T> {
+    // Storage: unknown [0x3a7472616e73616374696f6e5f6c6576656c3a] (r:1 w:1)
+    // Storage: GlobalDisputes Winners (r:1 w:1)
+    // Storage: GlobalDisputes Outcomes (r:1 w:1)
+    // Storage: GlobalDisputes Locks (r:1 w:1)
+    // Storage: Balances Locks (r:1 w:1)
+    fn vote_on_outcome(_o: u32, v: u32) -> Weight {
```

Comment:
Why is  `_o` disregarded here?

Reply:
Good question. This seems more and more like a bug. Because I ran into it a couple of times like in the prediction market benchmark PR. Now for example, as I used the debug build for benchmarks and 2 steps and 2 repetitions I got many `_` underscores for the components. I really think there is some kind of bug in the benchmark calculation or it's because of insufficient steps and/or repetitions.

Reply:
For `reward_outcome_owner` I realised, that I need to benchmark multiple paths since with the current implementation it's easier to see now. I will do this tomorrow.